ingestion/tests/cli_e2e/test_cli_mysql.py

```diff
@@ -92,12 +92,12 @@ def expected_filtered_schema_excludes() -> int:
 
     @staticmethod
     def expected_filtered_table_includes() -> int:
-        return 49
+        return 50
```

Comment:
increasing count, because we have added a new table `query_entity` which is filtered